dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -216,6 +216,67 @@ public fun DataFrame.Companion.readSqlQuery(
     }
 }
 
+/**
+ * Converts the result of an SQL query to the DataFrame.
+ * Also, it could be a just name of the SQL table.
+ *
+ * NOTE: It should be a name of one of the existing SQL tables,
```

Comment:
Shouldn't this be part of the `@param [sqlQueryOrTableName]` part? Or at least specify that this is information regarding that parameter. It's a bit unclear to me now.